### ISSUE TITLE
Adopt/pass through to CouchDB's native /_prometheus endpoint?

### ISSUE THREAD

**Issue by gesellix** [gesellix/couchdb-prometheus-exporter#104]
As of [CouchDB 3.2](https://blog.couchdb.org/2021/10/12/3-2-0/), there is native support for Prometheus metrics, which should more or less match our exporter's metrics. See details at https://docs.couchdb.org/en/latest/api/server/common.html#api-server-prometheus and https://github.com/apache/couchdb/issues/3377.

-> consider transforming this exporter to be
- deprecated for CouchDB >= 3.2
- label-compatible with CouchDB 3.2 `/_prometheus`
- pass-through for metrics being a available upstream (in the assumption that network traffic/latency can be reduced due to the reduced required requests)
- inspiration for even more metrics being exported by CouchDB (making this exporter obsolete for modern releases of CouchDB)
- ...?

**Comment by kripper** [comment]
Hi, I just installed grafana + prometheus + the new native CouchDB endpoint and tested the 3 published CouchDB dashboards, which obviously don't work because the metric names differ, so I was thinking in translating the Dashboards.
Do you maybe have a list of your metric names and their native equivalent name?

**Comment by gesellix** [comment]
Hi @kripper, no I didn't create an overview/mapping about the metrics, yet. Any help would be much appreciated!

**Comment by unlikelyzero** [comment]
@kripper any progress? I'm about to start

**Comment by kripper** [comment]
Hi, i tried 3 templates and fixed them a little to get a http status graph working and also to get the hosts list.
I believe it would be better to create a new template based on the existing ones.

Why would you still want to use the exporter instead of improving and using the native implementation?

**Comment by unlikelyzero** [comment]
> Hi, i tried 3 templates and fixed them a little to get a http status graph working and also to get the hosts list.
> I believe it would be better to create a new template based on the existing ones.
> 
> Why would you still want to use the exporter instead of improving and using the native implementation?

I personally haven't used the exporter. We just got the native one installed earlier today

**Comment by gesellix** [comment]
> Why would you still want to use the exporter instead of improving and using the native implementation?

As far as I understood, the native implementation provides metrics on node-level, so you'll have to scrape each node. That's without saying it's better or worse compared to an intermediate process like the exporter.

Could anyone of you who already use the native metrics provide a list or a scraped example of the actual metrics? I didn't find any time, yet, to set it up myself. I'd be interested in comparing the actual metrics and prepare the exporter to ease migration to the native metrics. I don't know if there's some kind of middle ground where the exporter makes sense, because it provides specific metrics the Apache CouchDB maintainers aren't willing to implement.

In the long term, I would assume that everyone is going to migrate to the native metrics, which would essentially make this exporter obsolete for modern CouchDB releases. I'm aware of CouchDB 1.x users, so I'll keep on maintaining the exporter for those.

So, to summarize: if anyone could provide a list of actual metrics from the native endpoint, please share them here. That would be much appreciated!

**Comment by unlikelyzero** [comment]
@gesellix 
`curl localhost:5984/_node/_local/_prometheus`

```bash
# TYPE couchdb_couch_log_requests_total counter
couchdb_couch_log_requests_total{level="alert"} 0
couchdb_couch_log_requests_total{level="critical"} 0
couchdb_couch_log_requests_total{level="debug"} 0
couchdb_couch_log_requests_total{level="emergency"} 0
couchdb_couch_log_requests_total{level="error"} 2
couchdb_couch_log_requests_total{level="info"} 15
couchdb_couch_log_requests_total{level="notice"} 7608
couchdb_couch_log_requests_total{level="warning"} 0
# TYPE couchdb_couch_replicator_changes_manager_deaths_total counter
couchdb_couch_replicator_changes_manager_deaths_total 0
# TYPE couchdb_couch_replicator_changes_queue_deaths_total counter
couchdb_couch_replicator_changes_queue_deaths_total 0
# TYPE couchdb_couch_replicator_changes_read_failures_total counter
couchdb_couch_replicator_changes_read_failures_total 0
# TYPE couchdb_couch_replicator_changes_reader_deaths_total counter
couchdb_couch_replicator_changes_reader_deaths_total 0
# TYPE couchdb_couch_replicator_checkpoints_failure_total counter
couchdb_couch_replicator_checkpoints_failure_total 0
# TYPE couchdb_couch_replicator_checkpoints_total counter
couchdb_couch_replicator_checkpoints_total 1
# TYPE couchdb_couch_replicator_cluster_is_stable gauge
couchdb_couch_replicator_cluster_is_stable 1
# TYPE couchdb_couch_replicator_connection_acquires_total counter
couchdb_couch_replicator_connection_acquires_total 21
# TYPE couchdb_couch_replicator_connection_closes_total counter
couchdb_couch_replicator_connection_closes_total 6
# TYPE couchdb_couch_replicator_connection_creates_total counter
couchdb_couch_replicator_connection_creates_total 6
# TYPE couchdb_couch_replicator_connection_owner_crashes_total counter
couchdb_couch_replicator_connection_owner_crashes_total 0
# TYPE couchdb_couch_replicator_connection_releases_total counter
couchdb_couch_replicator_connection_releases_total 27
# TYPE couchdb_couch_replicator_connection_worker_crashes_total counter
couchdb_couch_replicator_connection_worker_crashes_total 0
# TYPE couchdb_couch_replicator_db_scans_total counter
couchdb_couch_replicator_db_scans_total 1
# TYPE couchdb_couch_replicator_docs_completed_state_updates_total counter
couchdb_couch_replicator_docs_completed_state_updates_total 0
# TYPE couchdb_couch_replicator_docs_db_changes_total counter
couchdb_couch_replicator_docs_db_changes_total 0
# TYPE couchdb_couch_replicator_docs_dbs_created_total counter
couchdb_couch_replicator_docs_dbs_created_total 0
# TYPE couchdb_couch_replicator_docs_dbs_deleted_total counter
couchdb_couch_replicator_docs_dbs_deleted_total 0
# TYPE couchdb_couch_replicator_docs_dbs_found_total counter
couchdb_couch_replicator_docs_dbs_found_total 3
# TYPE couchdb_couch_replicator_docs_failed_state_updates_total counter
couchdb_couch_replicator_docs_failed_state_updates_total 0
# TYPE couchdb_couch_replicator_failed_starts_total counter
couchdb_couch_replicator_failed_starts_total 0
# TYPE couchdb_couch_replicator_jobs_adds_total counter
couchdb_couch_replicator_jobs_adds_total 1
# TYPE couchdb_couch_replicator_jobs_crashed gauge
couchdb_couch_replicator_jobs_crashed 0
# TYPE couchdb_couch_replicator_jobs_crashes_total counter
couchdb_couch_replicator_jobs_crashes_total 0
# TYPE couchdb_couch_replicator_jobs_duplicate_adds_total counter
couchdb_couch_replicator_jobs_duplicate_adds_total 0
# TYPE couchdb_couch_replicator_jobs_pending gauge
couchdb_couch_replicator_jobs_pending 0
# TYPE couchdb_couch_replicator_jobs_removes_total counter
couchdb_couch_replicator_jobs_removes_total 0
# TYPE couchdb_couch_replicator_jobs_running gauge
couchdb_couch_replicator_jobs_running 0
# TYPE couchdb_couch_replicator_jobs_starts_total counter
couchdb_couch_replicator_jobs_starts_total 1
# TYPE couchdb_couch_replicator_jobs_stops_total counter
couchdb_couch_replicator_jobs_stops_total 0
# TYPE couchdb_couch_replicator_jobs_total gauge
couchdb_couch_replicator_jobs_total 1
# TYPE couchdb_couch_replicator_requests_total counter
couchdb_couch_replicator_requests_total 27
# TYPE couchdb_couch_replicator_responses_failure_total counter
couchdb_couch_replicator_responses_failure_total 0
# TYPE couchdb_couch_replicator_responses_total counter
couchdb_couch_replicator_responses_total 21
# TYPE couchdb_couch_replicator_stream_responses_failure_total counter
couchdb_couch_replicator_stream_responses_failure_total 0
# TYPE couchdb_couch_replicator_stream_responses_total counter
couchdb_couch_replicator_stream_responses_total 0
# TYPE couchdb_couch_replicator_worker_deaths_total counter
couchdb_couch_replicator_worker_deaths_total 0
# TYPE couchdb_couch_replicator_workers_started_total counter
couchdb_couch_replicator_workers_started_total 4
# TYPE couchdb_auth_cache_requests_total counter
couchdb_auth_cache_requests_total 0
# TYPE couchdb_auth_cache_misses_total counter
couchdb_auth_cache_misses_total 0
# TYPE couchdb_collect_results_time_seconds summary
couchdb_collect_results_time_seconds{quantile="0.5"} 0.0
couchdb_collect_results_time_seconds{quantile="0.75"} 0.0
couchdb_collect_results_time_seconds{quantile="0.9"} 0.0
couchdb_collect_results_time_seconds{quantile="0.95"} 0.0
couchdb_collect_results_time_seconds{quantile="0.99"} 0.0
couchdb_collect_results_time_seconds{quantile="0.999"} 0.0
couchdb_collect_results_time_seconds_sum 0.0
couchdb_collect_results_time_seconds_count 0
# TYPE couchdb_couch_server_lru_skip_total counter
couchdb_couch_server_lru_skip_total 0
# TYPE couchdb_database_purges_total counter
couchdb_database_purges_total 0
# TYPE couchdb_database_reads_total counter
couchdb_database_reads_total 32172
# TYPE couchdb_database_writes_total counter
couchdb_database_writes_total 891
# TYPE couchdb_db_open_time_seconds summary
couchdb_db_open_time_seconds{quantile="0.5"} 0.0
couchdb_db_open_time_seconds{quantile="0.75"} 0.0
couchdb_db_open_time_seconds{quantile="0.9"} 0.0
couchdb_db_open_time_seconds{quantile="0.95"} 0.0
couchdb_db_open_time_seconds{quantile="0.99"} 0.0
couchdb_db_open_time_seconds{quantile="0.999"} 0.0
couchdb_db_open_time_seconds_sum 0.0
couchdb_db_open_time_seconds_count 0
# TYPE couchdb_dbinfo_seconds summary
couchdb_dbinfo_seconds{quantile="0.5"} 0.0
couchdb_dbinfo_seconds{quantile="0.75"} 0.0
couchdb_dbinfo_seconds{quantile="0.9"} 0.0
couchdb_dbinfo_seconds{quantile="0.95"} 0.0
couchdb_dbinfo_seconds{quantile="0.99"} 0.0
couchdb_dbinfo_seconds{quantile="0.999"} 0.0
couchdb_dbinfo_seconds_sum 0.0
couchdb_dbinfo_seconds_count 0
# TYPE couchdb_document_inserts_total counter
couchdb_document_inserts_total 19
# TYPE couchdb_document_purges_failure_total counter
couchdb_document_purges_failure_total 0
# TYPE couchdb_document_purges_success_total counter
couchdb_document_purges_success_total 0
# TYPE couchdb_document_purges_total_total counter
couchdb_document_purges_total_total 0
# TYPE couchdb_document_writes_total counter
couchdb_document_writes_total 890
# TYPE couchdb_httpd_aborted_requests_total counter
couchdb_httpd_aborted_requests_total 0
# TYPE couchdb_httpd_all_docs_timeouts_total counter
couchdb_httpd_all_docs_timeouts_total 0
# TYPE couchdb_httpd_bulk_docs_seconds summary
couchdb_httpd_bulk_docs_seconds{quantile="0.5"} 0.0
couchdb_httpd_bulk_docs_seconds{quantile="0.75"} 0.0
couchdb_httpd_bulk_docs_seconds{quantile="0.9"} 0.0
couchdb_httpd_bulk_docs_seconds{quantile="0.95"} 0.0
couchdb_httpd_bulk_docs_seconds{quantile="0.99"} 0.0
couchdb_httpd_bulk_docs_seconds{quantile="0.999"} 0.0
couchdb_httpd_bulk_docs_seconds_sum 0.0
couchdb_httpd_bulk_docs_seconds_count 0
# TYPE couchdb_httpd_bulk_requests_total counter
couchdb_httpd_bulk_requests_total 1
# TYPE couchdb_httpd_clients_requesting_changes_total counter
couchdb_httpd_clients_requesting_changes_total 3
# TYPE couchdb_httpd_explain_timeouts_total counter
couchdb_httpd_explain_timeouts_total 0
# TYPE couchdb_httpd_find_timeouts_total counter
couchdb_httpd_find_timeouts_total 0
# TYPE couchdb_httpd_partition_all_docs_requests_total counter
couchdb_httpd_partition_all_docs_requests_total 0
# TYPE couchdb_httpd_partition_all_docs_timeouts_total counter
couchdb_httpd_partition_all_docs_timeouts_total 0
# TYPE couchdb_httpd_partition_explain_requests_total counter
couchdb_httpd_partition_explain_requests_total 0
# TYPE couchdb_httpd_partition_explain_timeouts_total counter
couchdb_httpd_partition_explain_timeouts_total 0
# TYPE couchdb_httpd_partition_find_requests_total counter
couchdb_httpd_partition_find_requests_total 0
# TYPE couchdb_httpd_partition_find_timeouts_total counter
couchdb_httpd_partition_find_timeouts_total 0
# TYPE couchdb_httpd_partition_view_requests_total counter
couchdb_httpd_partition_view_requests_total 0
# TYPE couchdb_httpd_partition_view_timeouts_total counter
couchdb_httpd_partition_view_timeouts_total 0
# TYPE couchdb_httpd_purge_requests_total counter
couchdb_httpd_purge_requests_total 0
# TYPE couchdb_httpd_requests_total counter
couchdb_httpd_requests_total 7587
# TYPE couchdb_httpd_temporary_view_reads_total counter
couchdb_httpd_temporary_view_reads_total 0
# TYPE couchdb_httpd_view_reads_total counter
couchdb_httpd_view_reads_total 1
# TYPE couchdb_httpd_view_timeouts_total counter
couchdb_httpd_view_timeouts_total 0
# TYPE couchdb_httpd_request_methods counter
couchdb_httpd_request_methods{method="COPY"} 0
couchdb_httpd_request_methods{method="DELETE"} 3
couchdb_httpd_request_methods{method="GET"} 6352
couchdb_httpd_request_methods{method="HEAD"} 0
couchdb_httpd_request_methods{method="OPTIONS"} 0
couchdb_httpd_request_methods{method="POST"} 383
couchdb_httpd_request_methods{method="PUT"} 886
couchdb_httpd_status_codes{code="200"} 5617
couchdb_httpd_status_codes{code="201"} 888
couchdb_httpd_status_codes{code="202"} 0
couchdb_httpd_status_codes{code="204"} 0
couchdb_httpd_status_codes{code="206"} 0
couchdb_httpd_status_codes{code="301"} 0
couchdb_httpd_status_codes{code="302"} 0
couchdb_httpd_status_codes{code="304"} 1062
couchdb_httpd_status_codes{code="400"} 0
couchdb_httpd_status_codes{code="401"} 3
couchdb_httpd_status_codes{code="403"} 0
couchdb_httpd_status_codes{code="404"} 11
couchdb_httpd_status_codes{code="405"} 0
couchdb_httpd_status_codes{code="406"} 0
couchdb_httpd_status_codes{code="409"} 1
couchdb_httpd_status_codes{code="412"} 0
couchdb_httpd_status_codes{code="413"} 0
couchdb_httpd_status_codes{code="414"} 0
couchdb_httpd_status_codes{code="415"} 0
couchdb_httpd_status_codes{code="416"} 0
couchdb_httpd_status_codes{code="417"} 0
couchdb_httpd_status_codes{code="500"} 0
couchdb_httpd_status_codes{code="501"} 0
couchdb_httpd_status_codes{code="503"} 0
# TYPE couchdb_io_queue_search_total counter
couchdb_io_queue_search_total 0
# TYPE couchdb_io_queue2_search_count_total counter
couchdb_io_queue2_search_count_total 0
# TYPE couchdb_local_document_writes_total counter
couchdb_local_document_writes_total 6
# TYPE couchdb_mrview_emits_total counter
couchdb_mrview_emits_total 3
# TYPE couchdb_mrview_map_doc_total counter
couchdb_mrview_map_doc_total 3
# TYPE couchdb_open_databases_total counter
couchdb_open_databases_total 4
# TYPE couchdb_open_os_files_total counter
couchdb_open_os_files_total 4
# TYPE couchdb_query_server_vdu_process_time_seconds summary
couchdb_query_server_vdu_process_time_seconds{quantile="0.5"} 0.0
couchdb_query_server_vdu_process_time_seconds{quantile="0.75"} 0.0
couchdb_query_server_vdu_process_time_seconds{quantile="0.9"} 0.0
couchdb_query_server_vdu_process_time_seconds{quantile="0.95"} 0.0
couchdb_query_server_vdu_process_time_seconds{quantile="0.99"} 0.0
couchdb_query_server_vdu_process_time_seconds{quantile="0.999"} 0.0
couchdb_query_server_vdu_process_time_seconds_sum 0.0
couchdb_query_server_vdu_process_time_seconds_count 0
# TYPE couchdb_query_server_vdu_rejects_total counter
couchdb_query_server_vdu_rejects_total 0
# TYPE couchdb_request_time_seconds summary
couchdb_request_time_seconds{quantile="0.5"} 0.0
couchdb_request_time_seconds{quantile="0.75"} 0.0
couchdb_request_time_seconds{quantile="0.9"} 0.0
couchdb_request_time_seconds{quantile="0.95"} 0.0
couchdb_request_time_seconds{quantile="0.99"} 0.0
couchdb_request_time_seconds{quantile="0.999"} 0.0
couchdb_request_time_seconds_sum 0.0
couchdb_request_time_seconds_count 0
# TYPE couchdb_ddoc_cache_requests_total counter
couchdb_ddoc_cache_requests_total 33
# TYPE couchdb_ddoc_cache_requests_failures_total counter
couchdb_ddoc_cache_requests_failures_total 17
# TYPE couchdb_ddoc_cache_requests_recovery_total counter
couchdb_ddoc_cache_requests_recovery_total 0
# TYPE couchdb_dreyfus_httpd_search_seconds summary
couchdb_dreyfus_httpd_search_seconds{quantile="0.5"} 0.0
couchdb_dreyfus_httpd_search_seconds{quantile="0.75"} 0.0
couchdb_dreyfus_httpd_search_seconds{quantile="0.9"} 0.0
couchdb_dreyfus_httpd_search_seconds{quantile="0.95"} 0.0
couchdb_dreyfus_httpd_search_seconds{quantile="0.99"} 0.0
couchdb_dreyfus_httpd_search_seconds{quantile="0.999"} 0.0
couchdb_dreyfus_httpd_search_seconds_sum 0.0
couchdb_dreyfus_httpd_search_seconds_count 0
# TYPE couchdb_dreyfus_index_await_seconds summary
couchdb_dreyfus_index_await_seconds{quantile="0.5"} 0.0
couchdb_dreyfus_index_await_seconds{quantile="0.75"} 0.0
couchdb_dreyfus_index_await_seconds{quantile="0.9"} 0.0
couchdb_dreyfus_index_await_seconds{quantile="0.95"} 0.0
couchdb_dreyfus_index_await_seconds{quantile="0.99"} 0.0
couchdb_dreyfus_index_await_seconds{quantile="0.999"} 0.0
couchdb_dreyfus_index_await_seconds_sum 0.0
couchdb_dreyfus_index_await_seconds_count 0
# TYPE couchdb_dreyfus_index_group1_seconds summary
couchdb_dreyfus_index_group1_seconds{quantile="0.5"} 0.0
couchdb_dreyfus_index_group1_seconds{quantile="0.75"} 0.0
couchdb_dreyfus_index_group1_seconds{quantile="0.9"} 0.0
couchdb_dreyfus_index_group1_seconds{quantile="0.95"} 0.0
couchdb_dreyfus_index_group1_seconds{quantile="0.99"} 0.0
couchdb_dreyfus_index_group1_seconds{quantile="0.999"} 0.0
couchdb_dreyfus_index_group1_seconds_sum 0.0
couchdb_dreyfus_index_group1_seconds_count 0
# TYPE couchdb_dreyfus_index_group2_seconds summary
couchdb_dreyfus_index_group2_seconds{quantile="0.5"} 0.0
couchdb_dreyfus_index_group2_seconds{quantile="0.75"} 0.0
couchdb_dreyfus_index_group2_seconds{quantile="0.9"} 0.0
couchdb_dreyfus_index_group2_seconds{quantile="0.95"} 0.0
couchdb_dreyfus_index_group2_seconds{quantile="0.99"} 0.0
couchdb_dreyfus_index_group2_seconds{quantile="0.999"} 0.0
couchdb_dreyfus_index_group2_seconds_sum 0.0
couchdb_dreyfus_index_group2_seconds_count 0
# TYPE couchdb_dreyfus_index_info_seconds summary
couchdb_dreyfus_index_info_seconds{quantile="0.5"} 0.0
couchdb_dreyfus_index_info_seconds{quantile="0.75"} 0.0
couchdb_dreyfus_index_info_seconds{quantile="0.9"} 0.0
couchdb_dreyfus_index_info_seconds{quantile="0.95"} 0.0
couchdb_dreyfus_index_info_seconds{quantile="0.99"} 0.0
couchdb_dreyfus_index_info_seconds{quantile="0.999"} 0.0
couchdb_dreyfus_index_info_seconds_sum 0.0
couchdb_dreyfus_index_info_seconds_count 0
# TYPE couchdb_dreyfus_index_search_seconds summary
couchdb_dreyfus_index_search_seconds{quantile="0.5"} 0.0
couchdb_dreyfus_index_search_seconds{quantile="0.75"} 0.0
couchdb_dreyfus_index_search_seconds{quantile="0.9"} 0.0
couchdb_dreyfus_index_search_seconds{quantile="0.95"} 0.0
couchdb_dreyfus_index_search_seconds{quantile="0.99"} 0.0
couchdb_dreyfus_index_search_seconds{quantile="0.999"} 0.0
couchdb_dreyfus_index_search_seconds_sum 0.0
couchdb_dreyfus_index_search_seconds_count 0
# TYPE couchdb_dreyfus_rpc_group1_seconds summary
couchdb_dreyfus_rpc_group1_seconds{quantile="0.5"} 0.0
couchdb_dreyfus_rpc_group1_seconds{quantile="0.75"} 0.0
couchdb_dreyfus_rpc_group1_seconds{quantile="0.9"} 0.0
couchdb_dreyfus_rpc_group1_seconds{quantile="0.95"} 0.0
couchdb_dreyfus_rpc_group1_seconds{quantile="0.99"} 0.0
couchdb_dreyfus_rpc_group1_seconds{quantile="0.999"} 0.0
couchdb_dreyfus_rpc_group1_seconds_sum 0.0
couchdb_dreyfus_rpc_group1_seconds_count 0
# TYPE couchdb_dreyfus_rpc_group2_seconds summary
couchdb_dreyfus_rpc_group2_seconds{quantile="0.5"} 0.0
couchdb_dreyfus_rpc_group2_seconds{quantile="0.75"} 0.0
couchdb_dreyfus_rpc_group2_seconds{quantile="0.9"} 0.0
couchdb_dreyfus_rpc_group2_seconds{quantile="0.95"} 0.0
couchdb_dreyfus_rpc_group2_seconds{quantile="0.99"} 0.0
couchdb_dreyfus_rpc_group2_seconds{quantile="0.999"} 0.0
couchdb_dreyfus_rpc_group2_seconds_sum 0.0
couchdb_dreyfus_rpc_group2_seconds_count 0
# TYPE couchdb_dreyfus_rpc_info_seconds summary
couchdb_dreyfus_rpc_info_seconds{quantile="0.5"} 0.0
couchdb_dreyfus_rpc_info_seconds{quantile="0.75"} 0.0
couchdb_dreyfus_rpc_info_seconds{quantile="0.9"} 0.0
couchdb_dreyfus_rpc_info_seconds{quantile="0.95"} 0.0
couchdb_dreyfus_rpc_info_seconds{quantile="0.99"} 0.0
couchdb_dreyfus_rpc_info_seconds{quantile="0.999"} 0.0
couchdb_dreyfus_rpc_info_seconds_sum 0.0
couchdb_dreyfus_rpc_info_seconds_count 0
# TYPE couchdb_dreyfus_rpc_search_seconds summary
couchdb_dreyfus_rpc_search_seconds{quantile="0.5"} 0.0
couchdb_dreyfus_rpc_search_seconds{quantile="0.75"} 0.0
couchdb_dreyfus_rpc_search_seconds{quantile="0.9"} 0.0
couchdb_dreyfus_rpc_search_seconds{quantile="0.95"} 0.0
couchdb_dreyfus_rpc_search_seconds{quantile="0.99"} 0.0
couchdb_dreyfus_rpc_search_seconds{quantile="0.999"} 0.0
couchdb_dreyfus_rpc_search_seconds_sum 0.0
couchdb_dreyfus_rpc_search_seconds_count 0
# TYPE couchdb_fabric_doc_update_errors_total counter
couchdb_fabric_doc_update_errors_total 1
# TYPE couchdb_fabric_doc_update_mismatched_errors_total counter
couchdb_fabric_doc_update_mismatched_errors_total 0
# TYPE couchdb_fabric_doc_update_write_quorum_errors_total counter
couchdb_fabric_doc_update_write_quorum_errors_total 0
# TYPE couchdb_fabric_open_shard_timeouts_total counter
couchdb_fabric_open_shard_timeouts_total 0
# TYPE couchdb_fabric_read_repairs_failures_total counter
couchdb_fabric_read_repairs_failures_total 0
# TYPE couchdb_fabric_read_repairs_total counter
couchdb_fabric_read_repairs_total 0
# TYPE couchdb_fabric_worker_timeouts_total counter
couchdb_fabric_worker_timeouts_total 0
# TYPE couchdb_global_changes_db_writes_total counter
couchdb_global_changes_db_writes_total 0
# TYPE couchdb_global_changes_event_doc_conflict_total counter
couchdb_global_changes_event_doc_conflict_total 0
# TYPE couchdb_global_changes_listener_pending_updates gauge
couchdb_global_changes_listener_pending_updates 0
# TYPE couchdb_global_changes_rpcs_total counter
couchdb_global_changes_rpcs_total 0
# TYPE couchdb_global_changes_server_pending_updates gauge
couchdb_global_changes_server_pending_updates 0
# TYPE couchdb_mango_docs_examined_total counter
couchdb_mango_docs_examined_total 24101
# TYPE couchdb_mango_evaluate_selector_total counter
couchdb_mango_evaluate_selector_total 24101
# TYPE couchdb_mango_query_invalid_index_total counter
couchdb_mango_query_invalid_index_total 0
# TYPE couchdb_mango_query_time_seconds summary
couchdb_mango_query_time_seconds{quantile="0.5"} 0.0
couchdb_mango_query_time_seconds{quantile="0.75"} 0.0
couchdb_mango_query_time_seconds{quantile="0.9"} 0.0
couchdb_mango_query_time_seconds{quantile="0.95"} 0.0
couchdb_mango_query_time_seconds{quantile="0.99"} 0.0
couchdb_mango_query_time_seconds{quantile="0.999"} 0.0
couchdb_mango_query_time_seconds_sum 0.0
couchdb_mango_query_time_seconds_count 0
# TYPE couchdb_mango_quorum_docs_examined_total counter
couchdb_mango_quorum_docs_examined_total 0
# TYPE couchdb_mango_results_returned_total counter
couchdb_mango_results_returned_total 230
# TYPE couchdb_mango_too_many_docs_scanned_total counter
couchdb_mango_too_many_docs_scanned_total 16
# TYPE couchdb_mango_unindexed_queries_total counter
couchdb_mango_unindexed_queries_total 16
# TYPE couchdb_mem3_shard_cache_eviction_total counter
couchdb_mem3_shard_cache_eviction_total 4
# TYPE couchdb_mem3_shard_cache_hit_total counter
couchdb_mem3_shard_cache_hit_total 128089
# TYPE couchdb_mem3_shard_cache_miss_total counter
couchdb_mem3_shard_cache_miss_total 1760
# TYPE couchdb_pread_exceed_eof_total counter
couchdb_pread_exceed_eof_total 0
# TYPE couchdb_pread_exceed_limit_total counter
couchdb_pread_exceed_limit_total 0
# TYPE couchdb_rexi_buffered_total counter
couchdb_rexi_buffered_total 0
# TYPE couchdb_rexi_down_total counter
couchdb_rexi_down_total 0
# TYPE couchdb_rexi_dropped_total counter
couchdb_rexi_dropped_total 0
# TYPE couchdb_rexi_streams_timeout_total counter
couchdb_rexi_streams_timeout_total{stage="init_stream"} 0
# TYPE couchdb_rexi_streams_timeout_stream_total counter
couchdb_rexi_streams_timeout_stream_total 0
100 23531  100 23531    0     0  5744k      0 --:--:-- --:--:-- --:--:-- 5744k
xi_streams_timeout_wait_for_ack_total counter
couchdb_rexi_streams_timeout_wait_for_ack_total 0
# TYPE couchdb_uptime_seconds counter
couchdb_uptime_seconds 88322
# TYPE couchdb_erlang_memory_bytes gauge
couchdb_erlang_memory_bytes{memory_type="total"} 66254208
couchdb_erlang_memory_bytes{memory_type="processes"} 20488624
couchdb_erlang_memory_bytes{memory_type="processes_used"} 20487368
couchdb_erlang_memory_bytes{memory_type="system"} 45765584
couchdb_erlang_memory_bytes{memory_type="atom"} 639193
couchdb_erlang_memory_bytes{memory_type="atom_used"} 638789
couchdb_erlang_memory_bytes{memory_type="binary"} 880064
couchdb_erlang_memory_bytes{memory_type="code"} 13124163
couchdb_erlang_memory_bytes{memory_type="ets"} 2012480
# TYPE couchdb_erlang_gc_collections_total counter
couchdb_erlang_gc_collections_total 3050991
# TYPE couchdb_erlang_gc_words_reclaimed_total counter
couchdb_erlang_gc_words_reclaimed_total 15052172711
# TYPE couchdb_erlang_context_switches_total counter
couchdb_erlang_context_switches_total 10128404
# TYPE couchdb_erlang_reductions_total counter
couchdb_erlang_reductions_total 19267871995
# TYPE couchdb_erlang_processes gauge
couchdb_erlang_processes 402
# TYPE couchdb_erlang_process_limit gauge
couchdb_erlang_process_limit 262144
# TYPE couchdb_erlang_io_recv_bytes_total counter
couchdb_erlang_io_recv_bytes_total 9025874
# TYPE couchdb_erlang_io_sent_bytes_total counter
couchdb_erlang_io_sent_bytes_total 463004964
# TYPE couchdb_erlang_message_queues gauge
couchdb_erlang_message_queues 0
# TYPE couchdb_erlang_message_queue_min gauge
couchdb_erlang_message_queue_min 0
# TYPE couchdb_erlang_message_queue_max gauge
couchdb_erlang_message_queue_max 0
# TYPE couchdb_erlang_scheduler_queues gauge
couchdb_erlang_scheduler_queues 0
# TYPE couchdb_erlang_dirty_cpu_scheduler_queues gauge
couchdb_erlang_dirty_cpu_scheduler_queues 0
# TYPE couchdb_erlang_memory_bytes gauge
couchdb_erlang_memory_bytes{memory_type="total"} 66259920
couchdb_erlang_memory_bytes{memory_type="processes"} 20494288
couchdb_erlang_memory_bytes{memory_type="processes_used"} 20493032
couchdb_erlang_memory_bytes{memory_type="system"} 45765632
couchdb_erlang_memory_bytes{memory_type="atom"} 639193
couchdb_erlang_memory_bytes{memory_type="atom_used"} 638789
couchdb_erlang_memory_bytes{memory_type="binary"} 880184
couchdb_erlang_memory_bytes{memory_type="code"} 13124163
couchdb_erlang_memory_bytes{memory_type="ets"} 2012480
# TYPE couchdb_erlang_gc_collections_total counter
couchdb_erlang_gc_collections_total 3050993
# TYPE couchdb_erlang_gc_words_reclaimed_total counter
couchdb_erlang_gc_words_reclaimed_total 15052243468
# TYPE couchdb_erlang_context_switches_total counter
couchdb_erlang_context_switches_total 10128414
# TYPE couchdb_erlang_reductions_total counter
couchdb_erlang_reductions_total 19267907773
# TYPE couchdb_erlang_processes gauge
couchdb_erlang_processes 402
# TYPE couchdb_erlang_process_limit gauge
couchdb_erlang_process_limit 262144
# TYPE couchdb_erlang_ets_table gauge
couchdb_erlang_ets_table 192
```

**Comment by gesellix** [comment]
@unlikelyzero thanks a lot!
